flutter_local_notifications/example/lib/main.dart

```diff
@@ -1074,6 +1074,13 @@ class _HomePageState extends State<HomePage> {
       bot: true,
       icon: BitmapFilePathAndroidIcon(largeIconPath),
     );
+    final Person chef = Person(
+        name: 'Master Chef',
+        key: '3',
+        uri: 'tel:111222333444',
+        icon: ByteArrayAndroidIcon.fromBase64String(
+            await _base64encodedImage('http://placekitten.com/48/48')));
```

Comment:
Can this be made `https` instead of `http` as I recall that using `http` could lead to issues

Reply:
done